src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -328,7 +329,7 @@ protected void onException(final RuntimeException e)
         try
         {
             final int timeout = EXCEPTION_TIMEOUT * exceptionTimer;
-            this.setDelay(timeout);
+            this.setDelay(FMLEnvironment.production ? timeout : EXCEPTION_TIMEOUT);
```

Comment:
This is super annoying in dev so I nerfed it to zeroth level

Reply:
Agreed, good idea